src/runtime/pipeline/pipeline_function.cc

```diff
@@ -0,0 +1,101 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+#include "pipeline_function.h"
+
+#include <utility>
+#include <vector>
+namespace tvm {
+namespace runtime {
+/*!
+ * \brief Initialize the pipeline.
+ * \param modules The list of graph executor module.
+ * \param pipeline_conf The Dependency information of each graph executor module.
+ * \param mod_config The config information that generated by the export library function call.
+ */
+size_t PipelineFunction::PipelineInit(Array<Module> modules, const PipelineConfig& pipeline_config,
+                                      const ModuleConfig& mod_config) {
```

Comment:
Didn't understand the semantic of this function. After reviewing the rest parts, I realized that it's mostly due to the naming. I agree with @masahi that `graph_executors_` should not be named to `executor`. Maybe just call it `modules`, or `ordered_modules`.

Reply:
change to graph_modules_.